mandatory/3-function-output.js

```diff
@@ -1,14 +1,15 @@
-// Add comments to explain what this function does. You're meant to use Google!
+// It generates a random number using Math.random() and it multiplies by 10
```

Comment:
Math.random() returns a number greater than or equal to 0 and less than 1, so this returns a number greater than or equal to 0 and less than 10